File: CSV_Operator_POB2213/CSV_Operator_POB2213/Person.cs
using CsvHelper.Configuration.Attributes;

namespace CSV_Operator_POB2213
{
    public class Person
    {
        [Name("Name")] // Атрибуты как в шапке файла
        public string? Name { get; set; }
        [Name("Age")]
        public int Age { get; set; }
    }
}

File: Помелов М.А/TS/Lesson 4/person.ts
class Person {
    name: string;
    age!: number;

    constructor(name: string);
    constructor(name: string, age: number);
    constructor(name: string, age?: number) {
        if (age === undefined) {
            this.name = name;
            return;
        }


        this.name = name;
        this.age = age;
    }





    greet(): string {
        return `Привет, меня зовут ${this.name}, мне ${this.age} лет`
    }

    celebrateBirthday(): void {
        this.age += 1;
        console.log(`C днем Рождения ${this.name}! Теперь тебе ${this.age} лет!`);
    }

    error(): never {
        throw new Error('Ошибка!');
    }

}

const person: Person = new Person('Valeriy', 34);
console.log(person.greet());
person.celebrateBirthday();